### ISSUE TITLE
list_collections behaviour with no token

### ISSUE THREAD

**Issue by steffilazerte** [BirdStudiesCanada/NatureCountsAPI#25]
When I access `list_collections` without a token, some of the collections (e.g., ABATLAS1 and ABATLAS2 among others) return `yes` in the `access` column.

Since users must have a token to download the data, this is a bit misleading, and I suspect a hold over from when users weren't required to actually have a token to download the public collections (?).

**Comment by pmorrill** [comment]
Good point. These are collections that have public access, but since we require a token to get_data, this information is not very valuable. Probably, if there is no token in the request, we should not include the 'access' field in the response?

**Comment by steffilazerte** [comment]
Either that or `no` (as in no access). I can easily omit the field in the R client, but it might be better (for consistency) to do so in the API.
